addons/website_event/controllers/main.py

```diff
@@ -1,24 +1,21 @@
 # -*- coding: utf-8 -*-
 
 import time
-import werkzeug.urls
+import werkzeug
 from datetime import datetime, timedelta
 from dateutil.relativedelta import relativedelta
 
-from openerp import http
-from openerp import tools, SUPERUSER_ID
+from openerp import http, _
 from openerp.addons.website.models.website import slug
```

Comment:
organize import lines in alphabetical order like

imports of external python lib
imports of openerp
imports of Odoo Modules